README.md

```diff
@@ -146,6 +146,15 @@ The pulsar-manager can monitor topics and subscriptions.
    * Password: `pulsar`  
    * Pulsar manager website: http://localhost:9527/
 
+4. Add Pulsar standalone environment.
+    You can choose any name you would like for the Environment Name. The default Service URL is dependent on how you set up pulsar-manager and pulsar.
+    * Pulsar standalone with pulsar-manager docker image:
+    	Modify conf/standalone.conf to change `bindAddress=127.0.0.1`. Then, the Service URL is `http://host.docker.internal:8080`
+    * Pulsar standalone with pulsar-manager docker image:
+	By default, `http://0.0.0.0:8080`
+    * Pulsar and pulsar-manager docker image:
+	???
```

Comment:
I don't think it is appropriate to add content here. The following list refers to the starting sequence under the container. It may not be convenient for the user to modify the standalone.conf configuration file and restart the service for the container. Sometimes we should be able to access this service directly through the name of the container. If not, I think it is more appropriate to modify the /etc/hosts file and add a record.

1. Start Pulsar standalone.
2. Build an environment.
3. Access Pulsar manager website.

Reply:
I see. The core issue is that there is no parity between the instructions here and the instructions found in https://pulsar.apache.org/docs/en/standalone/. Since the recommended course for standalone is to install locally without docker image, it seems that there should be a clear pathway for users following the linked steps to access the Pulsar manager site.

Reply:
Yes, thank you for reminding me. I think some explanations can be added here at https://pulsar.apache.org/docs/en/administration-pulsar-manager/ before install. For example, users can start pulsar-manager directly if they already have pulsar cluster. If there is no pulsar cluster, they can use a container to start the pulsar service or start standalone service. Are you interested in adding this instruction? Thanks. @shea-engagio

Reply:
Sure, I will do so in the next few hours. Thanks, @tuteng